stuff/library/shaders/programs/HSLBlendGPU.frag

```diff
@@ -114,13 +115,21 @@ void main( void )
   // Perform blending
   if (fg_alpha > 0.0 && bg_alpha > 0.0) {
     vec3 o_pix = SetLumSat(bhue ? fg_pix : bg_pix, bsat ? fg_pix : bg_pix, blum ? fg_pix : bg_pix);
-    gl_FragColor.rgb = mix(bg_pix, o_pix, balpha);
+    if (matte) {
+      gl_FragColor.rgb = mix(bg_pix, o_pix, balpha * fg_frag.a);
```

Comment:
fg_alpha defined earlier in [#L106](https://github.com/opentoonz/opentoonz/blob/b1bffa51fefb348c2c9984798ec9010187786d0a/stuff/library/shaders/programs/HSLBlendGPU.frag#L106) holds the same value of `balpha * fg_frag.a` already.